xml_converter/generators/proto_templates/node.proto

```diff
@@ -0,0 +1,274 @@
+syntax = "proto3";
+import "google/protobuf/any.proto";
+
+package Proto_Node;
+
+message Category {
+    bool default_visibility = 1;
+    string display_name = 2;
+    bool is_separator = 3;
+    string name = 4;
+    string tooltip_name = 5;
+    map <string, Category> children = 6;
+}
+
+message Icon {
+    fixed32 achievement_bit = 1;
```

Comment:
Lower indexes should be reserved for more commonly used items. I would expect the first few numbers to be things like `map_id`, and `position`.

Reply:
Reworked indexes